App.tsx

```diff
@@ -86,6 +116,64 @@ function GeolocationSwitch() {
   );
 }
 
+function GeolocationConfig({ onUpdated }) {
+  const [ distanceFilter, setDistanceFilter] = React.useState('');
+  const [ elasticityMultiplier, setElasticityMultiplier] = React.useState('');
+  const [ desiredAccuracy, setDesiredAccuracy] = React.useState('');
+
+  React.useEffect(() => {
+    const setInitialTrackingConfig = async () => {
+      const trackingConfig = await getTrackingConfig()
+
+      console.log(trackingConfig)
+
+      setDistanceFilter(trackingConfig.distanceFilter.toString())
+      setElasticityMultiplier(trackingConfig.elasticityMultiplier.toString())
+      setDesiredAccuracy(trackingConfig.desiredAccuracy.toString())
+
+    };
+    setInitialTrackingConfig();
+  }, []);
```

Comment:
Won't this be called each time we set a new value (without even pressing the save button) ?

Reply:
No because there is an empty [] in the dependency array. So it will be called only when the component is mounted.